framework/src/dm/Kconfig

```diff
@@ -0,0 +1,37 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config DM
+	bool "DM"
```

Comment:
Instead of DM, why don't you use Device Management?

Reply:
I will change it at the next commit not this time.
Because there are so many places in which tizen rt uses it.